politeiawww/api/www/v1/v1.go

```diff
@@ -379,23 +379,37 @@ type CensorshipRecord struct {
-	Username            string      `json:"username"`                      // Username of user who submitted proposal
-	PublicKey           string      `json:"publickey"`                     // Key used for signature.
-	Signature           string      `json:"signature"`                     // Signature of merkle root
-	Files               []File      `json:"files"`                         // Files that make up the proposal
-	NumComments         uint        `json:"numcomments"`                   // Number of comments on the proposal
-	Version             string      `json:"version"`                       // Record version
-	StatusChangeMessage string      `json:"statuschangemessage,omitempty"` // Message associated to the status change
-	PublishedAt         int64       `json:"publishedat,omitempty"`         // The timestamp of when the proposal has been published
-	CensoredAt          int64       `json:"censoredat,omitempty"`          // The timestamp of when the proposal has been censored
-	AbandonedAt         int64       `json:"abandonedat,omitempty"`         // The timestamp of when the proposal has been abandoned
+	Name                string       `json:"name"`                          // Suggested short proposal name
+	State               PropStateT   `json:"state"`                         // Current state of proposal
+	Status              PropStatusT  `json:"status"`                        // Current status of proposal
+	Timestamp           int64        `json:"timestamp"`                     // Last update of proposal
+	UserId              string       `json:"userid"`                        // ID of user who submitted proposal
+	Username            string       `json:"username"`                      // Username of user who submitted proposal
+	PublicKey           string       `json:"publickey"`                     // Key used for signature.
+	Signature           string       `json:"signature"`                     // Signature of merkle root
+	Files               []File       `json:"files"`                         // Files that make up the proposal
+	NumComments         uint         `json:"numcomments"`                   // Number of comments on the proposal
+	Version             string       `json:"version"`                       // Record version
+	StatusChangeMessage string       `json:"statuschangemessage,omitempty"` // Message associated to the status change
+	PublishedAt         int64        `json:"publishedat,omitempty"`         // The timestamp of when the proposal has been published
+	CensoredAt          int64        `json:"censoredat,omitempty"`          // The timestamp of when the proposal has been censored
+	AbandonedAt         int64        `json:"abandonedat,omitempty"`         // The timestamp of when the proposal has been abandoned
+	VoteSummary         *VoteSummary `json:"votesummary"`                   // Summary of current state of voting process
```

Comment:
Actually, I was wrong about this.  The vote summary will always be present since the a proposal will always have a `VoteStatus`.  Even unvetted proposals.

You can ignore my above comment about making it `omitempty`.  The reason for making it a pointer was so that we could omit it, but since that is no longer required, it doesn't have to be a pointer.

Reply:
We can add `omitempty` though to all of the fields in `VoteSummary` that will not always be present.  Bascially all of them except `VoteSummary.Status`.

Reply:
I was thinking about this a little more.  Instead of directly copying the `VoteStatusReply` we should take this opportunity to clean things up a bit.  This is what I was thinking:

```
type VoteSummary struct {
	Status           PropVoteStatusT    `json:"status"`                     // Vote status
	EligibleTickets  uint32             `json:"eligibletickets,omitempty"`  // Number of eligible tickets
	EndHeight        uint64             `json:"endheight,omitempty"`        // Vote end height
	BestBlock        uint64             `json:"bestblock,omitempty"`        // Current best block
	QuorumPercentage uint32             `json:"quorumpercentage,omitempty"` // Percent of eligible votes required for quorum
	PassPercentage   uint32             `json:"passpercentage,omitempty"`   // Percent of total votes required to pass
	Results          []VoteOptionResult `json:"optionsresult,omitempty"`    // Vote results
}
```

You agree?

Reply:
Yeah I agree, that makes sense.